server/pulp/server/lazy/alias.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+
+class AliasTable(object):
+    """
+    Represents a collection of Apache *Alias* directives.
+
+    :ivar table: A table of alias => path mappings.
+    :type table: dict
+    """
+
+    def __init__(self):
+        self.table = {}
+
+    def load(self):
+        """
+        Load the alias table.
+        Read .conf files in /etc/httpd/conf.d/ and build the table
+        using *Alias* directives.
+        """
+        root = '/etc/httpd/conf.d'
```

Comment:
It'd be nice if this was configurable in some way. I know we've got a lot standing in the way of Pulp running on non-RHEL systems, but I'd rather not add to the pile.